### PR TITLE
Update docker-compose.yml with frequently used environment variables …

### PR DESCRIPTION
**Pull request by iVegas**
…for php container

### CODE REVIEW

**Comment by iVegas** [comment]
closes #507 
closes #508

**Comment by csandanov** [comment]
Hi, why set log level to 0? We probably do need to change the default to `3`

**Comment by iVegas** [comment]
The `php` image already uses the default log level value if xdebug is enabled.
In case I want to change the log level I'll uncomment the line and set a different log level.
Unless xdebux connection is not configured by a developer default value is enough to help establish it. Once IDE listens to the web server correctly logging is no longer needed.
So frequently used values for xdebug logger level are `3` or `0`. That`s why I provided a different to the default value for the xdebug logging level.

File: docker-compose.yml [hunk 1]
```diff
@@ -33,12 +33,14 @@ services:
...
 #      PHP_XDEBUG_MODE: debug
 #      PHP_XDEBUG_MODE: profile
+#      PHP_XDEBUG_START_WITH_REQUEST: "yes"
 #      PHP_XDEBUG_USE_COMPRESSION: false
 #      PHP_IDE_CONFIG: serverName=my-ide
 #      PHP_XDEBUG_IDEKEY: "my-ide"
 #      PHP_XDEBUG_CLIENT_HOST: 172.17.0.1 # Linux
 #      PHP_XDEBUG_CLIENT_HOST: host.docker.internal # Docker 18.03+ Mac/Win
 #      PHP_XDEBUG_CLIENT_HOST: 10.0.75.1 # Windows, Docker < 18.03
+#      PHP_XDEBUG_LOG_LEVEL: 0
```
Comment (iVegas) [inline docker-compose.yml:43]: As an option, we can provide both options similarly to `PHP_XDEBUG_MODE` environment variable.
```suggestion
#      PHP_XDEBUG_LOG_LEVEL: 0
#      PHP_XDEBUG_LOG_LEVEL: 3
```

**Comment by csandanov** [comment]
> Unless xdebux connection is not configured by a developer default value is enough to help establish it.

The current default log level is 7 which information, I guess it makes sense to change it to 3 (warnings), change it directly in the `wodby/php` image, not just here through env vars

**Comment by iVegas** [comment]
Added related PR https://github.com/wodby/php/pull/184

File: docker-compose.yml (hunk 1 above)
Comment (iVegas) [reply docker-compose.yml:43]:
```suggestion
```

**Pull request closed without merging**